packages/eds-core-react/src/components/Button/tokens/contained.ts

```diff
@@ -18,55 +18,55 @@ const {
       danger__hover: { rgba: dangerHoverColor },
     },
   },
-
   shape: {
     button: { borderRadius: buttonBorderRadius },
   },
 } = tokens
 
 export const primary = mergeDeepRight(button, {
-  background: primaryColor,
+  background: `var(--eds-color-surface-button-primary-filled, ${primaryColor})`,
```

Comment:
These button experiments should be reverted before we can merge. We'll try and do this the other way around first using the override variables in the existing tokens to inject new tokens, and then figure out the proper pattern for implementing new tokens directly later

Reply:
Reverted ⏮️